lib/discourse_plugin_registry.rb

```diff
@@ -129,6 +139,8 @@ def self.define_filtered_register(register_name)
 
   define_filtered_register :custom_filter_mappings
 
+  define_filtered_register :reviewable_types
```

Comment:
I just realised that we have [Plugin::Instance#register_reviewable_type](https://github.com/discourse/discourse/blob/aa66917df66a477561c4e0a536ddbb9fecb5513b/lib/plugin/instance.rb#L891) so might not need this?

Reply:
I ran into an issue where if there are no plugins that register reviewable types, then `DiscoursePluginRegistry#reviewable_types_lookup` is never created. I figured it was better to always define it (even if we don't have any plugin reviewable types), rather than having to check if that method has been created or not before calling it.